integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystemOpts.java

```diff
@@ -93,12 +73,10 @@ public static AlluxioFuseFileSystemOpts create(
         "AlluxioFuseCliOpts should not be null for creating AlluxioFuseFileSystemOpts.");
     String alluxioPath = fuseCliOpts.getMountAlluxioPath().orElseGet(
         () -> conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH));
-    Optional<String> authPolicyCustomGroup = conf.isSet(PropertyKey.FUSE_AUTH_POLICY_CUSTOM_GROUP)
-        ? Optional.of(conf.getString(PropertyKey.FUSE_AUTH_POLICY_CUSTOM_GROUP))
-        : Optional.empty();
-    Optional<String> authPolicyCustomUser = conf.isSet(PropertyKey.FUSE_AUTH_POLICY_CUSTOM_USER)
-        ? Optional.of(conf.getString(PropertyKey.FUSE_AUTH_POLICY_CUSTOM_USER))
-        : Optional.empty();
+    Optional<String> authPolicyCustomGroup = Optional.ofNullable(
```

Comment:
The examples don't completely apply in our case because `conf.getString` will never return null. If it's not set, an exception will be thrown. So we have to check first whether it's set and `ofNullable` can't apply in here.